bot/seasons/christmas/hanukkah_embed

```diff
@@ -0,0 +1,110 @@
+from discord.ext import commands
+from discord import Embed
+import requests
+import datetime
+import logging
+
+
+log = logging.getLogger(__name__)
+
+HANUKKAH_DAYS = []
+HANUKKAH_MONTHS = []
+HANUKKAH_YEARS = []
```

Comment:
Check on the naming conventions. Capitalized for constants. 
Also, these variables can be initiated in `__init__` as well.

Reply:
okay